.clang-tidy

```diff
@@ -19,6 +19,7 @@ Checks: |
   -cppcoreguidelines-init-variables
   -cppcoreguidelines-pro-type-member-init
   -cppcoreguidelines-avoid-do-while
+  -cppcoreguidelines-avoid-non-const-global-variables
```

Comment:
With this we might be able to restore `examples` to the list of files we check...

Reply:
I wanted to see if `ImplementationFileExtensions` would filter out that root file since it uses a different extension, but it seems that CMake is still checking it..